docs/Configuration-Properties.md

```diff
@@ -37,11 +37,27 @@ The Enrollment Server uses the following public configuration properties:
 | `enrollment-server.inbox.enabled` | `true` | Publishing of Inbox endpoints can be enabled or disabled using this property. |
 | `enrollment-server.activation-spawn.enabled` | `false` | The activation spawn functionality can be enabled or disabled using this property. |
 
+## UserInfoProvider Configuration
+
+| Property                                                                          | Default   | Note                                                                                                                    |
+|-----------------------------------------------------------------------------------|-----------|-------------------------------------------------------------------------------------------------------------------------|
+| `enrollment-server.user-info.minimal-provider.enabled`                            | `false`   | Whether to register minimal claims provider. Must not be set to `true` when `rest-provider.enabled` set also to `true`. |
```

Comment:
We should decide, how we want to configure dev environment. Whether to enable minimal-provider or wait for user-data-store component.